src/serial.rs

```diff
@@ -478,6 +536,34 @@ mod tests {
         }
     }
 
+    #[derive(Default)]
+    struct ExampleSerialMetrics {
+        read_count: AtomicU64,
+        write_all_count: AtomicU64,
+        flush_count: AtomicU64,
+    }
+
+    // We define this for Arc<SerialExampleMetrics> because we expect to share a reference to
+    // the same object with other threads.
+    // TODO: We can use `Metric` from `vmm-sys-util` after it gets published. Also, can
+    // we make `vmm-sys-util` a dev-dependency (vs. a full build dependency) now that we
+    // added the `Trigger` abstraction and objects in this crate no longer have `EventFd`
+    // fields?
```

Comment:
:+1:

Reply:
changed `vmm-sys-util` to a dev-dependency